Microsoft.Toolkit.Win32/Microsoft.Toolkit.Win32.UI.Controls/WPF/WebView/Compatible/WebBrowserCompatibilityAdapter.cs

```diff
@@ -0,0 +1,89 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Navigation;
+using Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT;
+
+namespace Microsoft.Toolkit.Win32.UI.Controls.WPF.Compatible
+{
+    internal sealed class WebBrowserCompatibilityAdapter : WebBaseCompatibilityAdapter
+    {
+        private WebBrowser _browser = new WebBrowser();
+
+        private void Browser_Navigated(object sender, NavigationEventArgs e)
+        {
+            NavigationCompleted?.Invoke(sender, e);
+        }
+
+        private void Browser_Navigating(object sender, NavigatingCancelEventArgs e)
+        {
+            NavigationStarting?.Invoke(sender, e);
+            ContentLoading?.Invoke(sender, e);
+        }
+
+        public override Uri Source { get => _browser.Source; set => _browser.Source = value; }
+
+        public override bool CanGoBack => _browser.CanGoBack;
+
+        public override bool CanGoForward => _browser.CanGoForward;
+
+        public override FrameworkElement View => _browser;
+
+        public override event EventHandler<WebViewControlNavigationStartingEventArgs> NavigationStarting;
+
+        public override event EventHandler<WebViewControlContentLoadingEventArgs> ContentLoading;
+
+        public override event EventHandler<WebViewControlNavigationCompletedEventArgs> NavigationCompleted;
+
+        public override bool GoBack()
+        {
+            _browser.GoBack();
+            return true;
+        }
+
+        public override bool GoForward()
+        {
+            _browser.GoForward();
+            return true;
+        }
+
+        public override string InvokeScript(string scriptName)
+        {
+            return _browser.InvokeScript(scriptName)?.ToString();
```

Comment:
Can't always guarantee `object` will `ToString` as you expect. You can type check here to see if string and return that, otherwise best-effort `ToString` or throw a `NotSupportedException`.

Reply:
I'm on the fence about this kind of thing, generally.  Sometimes doing too much for developers obscures problems (or solutions) unnecessarily, making it hard to get to the root of things, but I'll defer to your judgement.

Reply:
The compatible thing here is to return `object`, no? String will be cast to object, and the developer can do the right thing.

Reply:
Aye...  good point.  I was trying to fit to an ascribed interface...